DEV/dev.db.sql

```diff
@@ -1 +0,0 @@
-\i ../create_db_ddl.sql
```

Comment:
Why have you deleted this?

Reply:
That must be an accident as I have moved cleared my client branch if you found any folder deleted.
Please do not merge I will correct my branch and then you can merge it. However please advise as I have worked on create and delete script only.

Thanks
~Shakir